drivers/gles3/storage/render_scene_buffers_gles3.h

```diff
@@ -103,9 +103,9 @@ class RenderSceneBuffersGLES3 : public RenderSceneBuffers {
 	virtual void configure(const RenderSceneBuffersConfiguration *p_config) override;
 	void configure_for_probe(Size2i p_size);
 
-	virtual void set_fsr_sharpness(float p_fsr_sharpness) override{};
-	virtual void set_texture_mipmap_bias(float p_texture_mipmap_bias) override{};
-	virtual void set_use_debanding(bool p_use_debanding) override{};
+	virtual void set_fsr_sharpness(float p_fsr_sharpness) override {}
```

Comment:
clang-format just fixed the missing space, but I took the opportunity to remove the unnecessary semicolon.

Reply:
BTW if anyone wants a challenge and try to finally write a script or find a clang-tidy hook that can remove useless semicolons, be my guest :D 
I've tried and failed a few times, but I get annoyed every time I find some in the codebase.

Reply:
I dunno about clang-tidy, but clang-format certainly has unnecessary semicolon checks. The ramifications of that are pretty wide-reaching, but I can probably make a draft PR for it